mgmt/src/processor/proc.rs

```diff
@@ -73,10 +77,18 @@ impl ConfigProcessor {
     pub(crate) fn new(frrmi: FrrMi) -> (Self, Sender<ConfigChannelRequest>) {
         debug!("Creating config processor...");
         let (tx, rx) = mpsc::channel(Self::CHANNEL_SIZE);
+
+        let Ok((connection, netlink, _)) = rtnetlink::new_connection() else {
+            panic!("failed to create connection");
```

Comment:
`panic!`?

Reply:
API stability during integration. Will fix in a follow on PR.